example/index.ts

```diff
@@ -0,0 +1,20 @@
+import { MongoSessionDaemon } from '../src/index'
+import { Db } from 'mongodb'
+
+const mongoUri = process.env.mongoUri as string
+const dbName = process.env.dbName as string
+const collectionName = process.env.collectionName as string
+
+const connectDB = async (dbName: string): Promise<Db> => {
+  const db = new MongoSessionDaemon(mongoUri)
+  const session = await db.getSession()
+  return await session.db(dbName)
```

Comment:
`session.db()` returns `Db` directly so this `await` is unnecessary.

Reply:
[fix unnecessary `await`](https://github.com/heineiuo/mongo-client-daemon/pull/1/commits/32aa0d4130ba3046ab7f444e513e58d505a2de6f)